cli/cmd/install.go

```diff
@@ -196,14 +197,35 @@ type InstallCmdHandler struct {
 	userInput        common.IUserInput
 }
 
+func isValidURL(chartURL string) bool {
```

Comment:
pls use the func from keptn go-utils https://github.com/keptn/go-utils/blob/ef732acee71f359e7c5a591a0927e489edd242a5/pkg/common/httputils/httputils.go#L63

Reply:
Ahh ok, I was not aware of that! Will do that. :+1: